src/pages/resetPassword.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import { BaseLayout } from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { USER_COOKIE } from '../constants';
+import { ErrorInfo } from '../interfaces';
+import { redirectTo } from './api/apiUtils';
+
```

Comment:
Import CsrfForm component and use in place of form element.

Reply:
done